<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>301</width>
      <height>401</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>nothing to display</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="User_input">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>451</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myriad Pro</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QplainTextEdit{
	background-color: rgb(255, 255, 255);
	padding: 10px; 
	border-radius: 20px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="plainText">
     <string>Start writing or Paste here...!</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>252</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="open_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
background-color :rgb(170, 170, 255);
color: white; 
border: none; 
padding: 10px; 
border-radius: 5px;
 }

</string>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Clear_all_button_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
background-color: rgb(255, 0, 0);
color: white; 
border: none; 
padding: 10px; 
border-radius: 5px;
 }</string>
       </property>
       <property name="text">
        <string>Clear All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Convert_button_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
background-color: #4CAF50; 
color: white; 
border: none; 
padding: 10px; 
border-radius: 5px; 
}</string>
       </property>
       <property name="text">
        <string>Convert</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="current_text_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>449</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { 
border: 2px dashed #2196F3; 
 border-radius: 30px; 
padding: 10px; 
}</string>
    </property>
    <property name="text">
     <string>Nothing to read ...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>-3</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>508</y>
      <width>761</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
 border-radius: 30px; 
background-color: #FFFFFF; 
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Play_button_2">
     <property name="geometry">
      <rect>
       <x>605</x>
       <y>20</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../assets/play.png</normaloff>../assets/play.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>34</width>
       <height>31</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Stop_button_2">
     <property name="geometry">
      <rect>
       <x>674</x>
       <y>20</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../assets/stop.png</normaloff>../assets/stop.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>39</width>
       <height>33</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Rewind_button_2">
     <property name="geometry">
      <rect>
       <x>709</x>
       <y>20</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../assets/rewind-button.png</normaloff>../assets/rewind-button.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>39</width>
       <height>33</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Pause_button_2">
     <property name="geometry">
      <rect>
       <x>639</x>
       <y>20</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../assets/pause.png</normaloff>../assets/pause.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>39</width>
       <height>33</height>
      </size>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_2">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>33</y>
       <width>461</width>
       <height>16</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
                border: 2px solid grey;
                border-radius: 5px;
                background-color: #FFFFFF;
            }
            QProgressBar::chunk {
                background-color: #2196F3;
                width: 10px;
            }
            QProgressBar::sub-page {
                background-image: url('ball.png');
            }</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>32</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>517</x>
       <y>32</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-- . --</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Save_button_2">
     <property name="geometry">
      <rect>
       <x>559</x>
       <y>21</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../assets/save.png</normaloff>../assets/save.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>34</width>
       <height>28</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
